static/app/views/dashboardsV2/widgetCard/chart.tsx

```diff
@@ -427,10 +427,9 @@ class WidgetCardChart extends Component<WidgetCardChartProps, State> {
             );
           }
 
-          const shouldColorOther =
-            widget.displayType === 'top_n' &&
-            timeseriesResults &&
-            timeseriesResults.length > 5;
+          const shouldColorOther = timeseriesResults
+            ?.map(series => series.seriesName)
+            .includes('Other');
```

Comment:
If you give your query an Alias, the seriesName is no longer `Other` which causes this to fail
![image](https://user-images.githubusercontent.com/83961295/167461661-6c870575-df4f-4832-bbd7-1df3b3f14ed4.png)

Also, maybe we should store `Other` in a constant if we are using it in this way

Reply:
Ah, good catch. Instead of just using `includes` I went with using a regex to match. It either matches on "Other" by itself, or "XYZ : Other" now

Series can have a different format with multi y-axes or multiple queries but we drop the "Other" series in those cases.

I also stored `Other` in a constant at the top of this file.